notebooks/Tutorial10_Geolocation_privacy_assessment.py

```diff
@@ -109,9 +102,27 @@
 plt.show
 # -
 
-# ## Run a geolocation privacy assessment
+# ## Density representation
+
+# The density representation is a way to visualize data using a heatmap. It provides a visual representation of the distribution of data points in a given area. In the case of the Porto taxi traces, the density representation shows the concentration of taxi traces in different regions of Porto.
+#
+# There are two common methods used to create a density representation: histogram and kernel density estimation.
+# - Histogram: In this method, the area is divided into a grid of cells, and the number of data points falling into each cell is counted. The count is then used to assign a color to each cell, creating a heatmap. Cells with a higher count will have a darker color, indicating a higher density of data points in that region.
+# - Kernel Density Estimation: This method uses a statistical technique to estimate the underlying probability density function of the data. It involves placing a kernel (a smooth, continuous function) on each data point and summing up the contributions of all kernels to create a smooth density estimate. The density estimate is then used to generate a heatmap, where regions with higher density have a darker color.
+#
+# Both methods provide a visual representation of the density of data points, but they differ in terms of how they estimate the density. Histograms are simpler and provide a more discrete representation, while kernel density estimation produces a smoother and continuous density estimate.
+#
+# Here is the density representation of all Porto taxi' traces:
+
+# ![Density representation of Porto' traces£](img/porto_density.png)
```

Comment:
I actually think that we should not include png here but something generated from the data. This has more its place in the doc (https://docs.octopize.io/docs/understanding/geolocation_privacy_assessment)